mono/eglib/glib.h

```diff
@@ -1,5 +1,19 @@
 #ifndef __GLIB_H
 #define __GLIB_H
+
+// Ask stdint.h for the full C99 features for CentOS 6 g++ 4.4.
+#if defined (__cplusplus) && __cplusplus <= 199711L // It is 1 for g++4.4 -std=c++0x.
```

Comment:
This should just be if cplusplus.
Android needs this too.

See
 $HOME/android-toolchain/toolchains/armeabi-v7a-clang/sysroot/usr/include/inttypes.h  $HOME/android-toolchain/toolchains/armeabi-v7a-clang/sysroot/usr/include/stdint.h